Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -120,6 +120,30 @@
       params:
         volume: 12
 
+- type: entity
+  name: Self Destruct
+  description: "Go out on your own terms, hell, even take them with you! kill them all, you know you want to! do it! (disclaimer: does not arm without power)"
```

```suggestion
  description: "Go out on your own terms!
```

Way too long and fluffy, also capitalisation.